### PR TITLE
add  kube_pod_status_ready_time metric

### PR DESCRIPTION
**Pull request by superbigsea**
**What this PR does / why we need it**:
add kube_pod_status_ready_time metric
 it can be used to monitor if the pod have long preparation_period

**How does this change affect the cardinality of KSM**: *(increases, decreases or does not change cardinality)*

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by fpetkovski** [comment]
Thanks for the contribution!

Could you also add a test case for the new metric as well as update the pod metric docs here: https://github.com/kubernetes/kube-state-metrics/blob/master/docs/pod-metrics.md

Thanks!

**Comment by superbigsea** [comment]
let me try
this is my friest time to do pr in github
thank you


> Thanks for the contribution!
> 
> Could you also add a test case for the new metric as well as update the pod metric docs here: https://github.com/kubernetes/kube-state-metrics/blob/master/docs/pod-metrics.md
> 
> Thanks!

**Comment by mrueg** [comment]
Let us know if you need any support with, you can reach us in #kube-state-metrics on Kubernetes' Slack.
https://app.slack.com/client/T09NY5SBT/CJJ529RUY

You need to sign the Kubernetes CLA as well.

> let me try
> this is my friest time to do pr in github
> thank you
> 
> > Thanks for the contribution!
> > Could you also add a test case for the new metric as well as update the pod metric docs here: https://github.com/kubernetes/kube-state-metrics/blob/master/docs/pod-metrics.md
> > Thanks!

File: docs/pod-metrics.md [hunk 1]
```diff
@@ -82,3 +82,4 @@ groups:
     annotations:
       summary: Pod {{labels.namespace}}/{{labels.pod}} block in Terminating state.
 ```
+
```
Comment (mrueg) [inline docs/pod-metrics.md:85]: Can you remove this line?

**Pull request closed without merging**